testing/unit_tests/test_rgb.py

```diff
@@ -9,23 +9,27 @@ def setup_function():
     create_tmp_dir()
 
 
-def test_save_rgb():
-    # Create image with shape HxWxDxC
-    image = [[[[0 for i in range(10)] for i in range(10)] for i in range(4)] for i in range(3)]
-    for i in range(3):
-        image[0][2][5][i] = 1
-        image[1][2][5][i+3] = 1
-        image[2][2][5][i + 6] = 1
-    image_n = np.array(image)
-    imed_visualize.save_color_labels(
-        gt_data=image_n,
-        binarize=False,
-        gt_filename=os.path.join(
-            __data_testing_dir__,
-            "derivatives/labels/sub-unf01/anat/sub-unf01_T2w_lesion-manual.nii.gz"),
-        output_filename=os.path.join(__tmp_dir__, "rgb_test.nii.gz"),
-        slice_axis=0
-    )
+# TODO: un-silence this test. This test was silenced because it was failing after introducing the copy of the header in
```

Comment:
If there is an issue ticket for this, it would be good practice to reference the issue # here

Reply:
thanks, done in https://github.com/ivadomed/ivadomed/pull/714/commits/a00a54fc5c701ba6f717168c2ed43a972230b051